### ISSUE TITLE
About LEO and other experiment implement details (like ablation study, conflicts study)

### ISSUE THREAD

**Issue by ligeng0197** [baiksung/L2F#3]
Hi David! I really appreciate what you did in this work, I believe this should be considered as a great progress in meta-learning.  I check your repository and find that the leo experiment and conflicts study mentioned in paper didn't show up here, would you mind adding them to this repository? That would be really helpful, thanks a lot.

p.s. salute to your hard work with only one 1080ti in all these experiments:)

**Comment by baiksung** [comment]
Hi,
Sorry for the late response. I've been occupied with other works.
And thank you for the interests in our work.
We are not planning to release other additional code any time soon as I just don't have enough time.
Just a few pointers to guide you though:

1. As for LEO experiments, We just added the attenuation part to [LEO official code](https://github.com/deepmind/leo) without changing hyperparameters or settings. If you are having trouble with porting to Tensorflow, let me know. I'm willing to help. Also, I'm planning to release a new code for TPAMI extended version of this paper, which includes MetaDataset results (in Tensorflow). So, please refer to that once we release them.

2. As for conflict study, I think we will release a separate code for this soon (within 1~2 months time frame) after code refactor. If you need to produce results earlier, please refer to the explanations in our paper and [26]. Again if you are having trouble, I'm willing to help.

[26] Santurkar et al. How Does Batch Normalization Help Optimization? 

P.S. Just to clarify: Each experiment was performed on a single 1080Ti. But, they were performed in parallel using a multiple of them. Using only a single GPU for the whole paper would be a nightmare. Regardless, thank you for your words.

Best,
Sungyong
